backend/service/auditsink/slack/slack_helper.go

```diff
@@ -0,0 +1,28 @@
+package slack
+
+import (
+	auditv1 "github.com/lyft/clutch/backend/api/audit/v1"
+	configv1 "github.com/lyft/clutch/backend/api/config/service/auditsink/slack/v1"
+	"github.com/lyft/clutch/backend/middleware"
+)
+
+// GetSlackOverrideText returns the custom slack message from the slack override config if the FullMethod matches
+// the serive and method in the audit event. Otherwise ok is false and an empty string is returned.
+func GetSlackOverrideText(override *configv1.Override, event *auditv1.RequestEvent) (string, bool) {
```

Comment:
Following the pattern for how we decide if we write an audit event to the slack sink. For example, we check if the filter specified in the config matches given values in the audit event (https://github.com/lyft/clutch/blob/main/backend/service/auditsink/slack/slack.go#L52 -> https://github.com/lyft/clutch/blob/main/backend/service/auditsink/sink.go#L19).

Similarly here, I check if the FullMethod in the slack override config matches the service and method in the audit event.